astrokat/simulate.py

```diff
@@ -17,7 +17,7 @@
 simobserver = ephem.Observer()
 
 _DEFAULT_SLEW_TIME = 45.0  # [sec]
-SIM_OVERHEAD = 3
+_SIM_OVERHEAD = 3           # [sec]
```

Comment:
Comments should be 2 spaces are variable.
```suggestion
_SIM_OVERHEAD = 3  # [sec]
```